compiler/rustc_data_structures/src/sip128.rs

```diff
@@ -1,5 +1,6 @@
 //! This is a copy of `core::hash::sip` adapted to providing 128 bit hashes.
 
+use rustc_serialize::int_overflow::{DebugStrictAdd, DebugStrictSub};
```

Comment:
Nit: could you add a comment explaining why we're using these operations in this module?